course/src/main/java/org/openedx/course/presentation/outline/CourseOutlineViewModel.kt

```diff
@@ -79,6 +85,9 @@ class CourseOutlineViewModel(
     val resumeBlockId: SharedFlow<String>
         get() = _resumeBlockId.asSharedFlow()
 
+    private val _canShowPLSBanner = mutableStateOf(false)
```

Comment:
We are not fetching the data anymore, flag is always false.

Reply:
updated